soc/xtensa/intel_apl_adsp/Kconfig.defconfig

```diff
@@ -34,9 +32,14 @@ config 2ND_LEVEL_INTERRUPTS
 config DYNAMIC_INTERRUPTS
 	default y
 
-endif
+config LOG
+	default y
 
-if LOG
+config LOG_PRINTK
+	default y
+
+config LOG_IMMEDIATE
+	default y
```

Comment:
How will this behave is an application does `CONFIG_LOG=n` in its `prj.conf` file. Wouldn't it cause warnings here fore these defaults, i.e. should they still be kept behind `if LOG`?

Reply:
Added LOG ifdefs, so I can build samples with LOG=n, like shell.